sync.js

```diff
@@ -479,9 +479,27 @@ class Sync extends events.EventEmitter {
           if (data && data['p2p-db'] && data['p2p-db'] !== 'kappa-osm') {
             return onerror(new Error('trying to sync this kappa-osm database with a ' + data['p2p-db'] + ' database!'))
           }
-          if (data && data.discoveryKey && opts.projectKey && data.discoveryKey !== discoKey) {
-            return onerror(new Error(`trying to sync two different projects (us=${discoKey}) (syncfile=${data.discoveryKey})`))
+
+          const intoDefaultProject = !opts.projectKey
+          const fileDiscoveryKey = data && data.discoveryKey
+
+          if (!fileDiscoveryKey) {
```

Comment:
A little bit stuck on this check, which accounts for when desktop exports a sync file and immediately imports it.

Sync files exported from a default project also don't have a discovery key. If we left this as is, custom projects could technically sync with sync files generated from default projects, which I don't think is the desired behavior.

Question becomes: how do we distinguish a sync file that was created via desktop export (especially from a custom project) vs. a sync file that was created from a default project?

cc @gmaclennan